contracts/infrastructure/ens/ArgentENSManager.sol

```diff
@@ -85,16 +84,23 @@ contract ArgentENSManager is IENSManager, Owned, Managed {
     }
 
     /**
-    * @notice Lets the manager assign an ENS subdomain of the root node to a target address.
-    * Registers both the forward and reverse ENS.
-    * @param _label The subdomain label.
-    * @param _owner The owner of the subdomain.
-    * @param _managerSignature The manager signature of the hash of _owner and _label.
-    */
+     * @inheritdoc IENSManager
+     */
     function register(string calldata _label, address _owner, bytes calldata _managerSignature) external override validateENSLabel(_label) {
         bytes32 signedHash = keccak256(abi.encodePacked("\x19Ethereum Signed Message:\n32", keccak256(abi.encodePacked(_owner, _label))));
         validateManagerSignature(signedHash, _managerSignature);
 
+        _register(_label, _owner);
+    }
+
+    /**
+     * @inheritdoc IENSManager
+     */
+    function register(string calldata _label, address _owner) external override onlyManager validateENSLabel(_label) {
```

Comment:
In `validateENSLabel` I believe it is not necessary to copy the bytes representation of the string to memory to get its length. We could just do:
``` 
modifier validateENSLabel(string memory _label) {
      require(bytes(_label).length != 0, "AEM: ENS label must be defined");
      _;
    }
```